system/ee/legacy/libraries/Template.php

```diff
@@ -1903,6 +1905,13 @@ public function process_tags()
 	 */
 	public function fetch_param($which, $default = FALSE)
 	{
+
+		if(in_array($which, $this->ignore_fetch)) {
+
+			return $this->tagparams[$which];
```

Comment:
This throws a PHP "undefined index" Notice. I setup a template with a basic channel entries loop with `title` and `url_title`.

Reply:
@obfuscode Fixed in latest commit.